### PR TITLE
lint in ci

### PR DESCRIPTION
**Pull request by KevinAnthony**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,8 +1,11 @@
-module github.com/KevinAnthony/bzen
+module github.com/kevinanthony/bzen
 
 go 1.16
 
 require (
+	github.com/json-iterator/go v1.1.11
 	github.com/pkg/errors v0.9.1
+	github.com/smartystreets/goconvey v1.6.4
+	github.com/stretchr/testify v1.3.0
```
Comment (KevinAnthony) [inline go.mod:9]: why?!?

File: http/client.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+	native "net/http"
+	"strings"
+
+	"github.com/kevinanthony/bzen/http/encoder"
+
+	"github.com/pkg/errors"
+)
+
+var (
+	errBadRequest = errors.New("bad request")
```
Comment (JeffKJCheng) [inline http/client.go:14]: --test line for discussion sake

Comment (KevinAnthony) [reply http/client.go:14]: no, your wrong

**Review by JeffKJCheng — APPROVED** [review]
Reviewed with Kevin on overall commits, types, base, mocks, and tests

**Pull request merged**